tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -63,7 +63,7 @@ def get(self, request, *args, **kwargs):
         """Handle get request and verify user is authorized to access kibana."""
         user = request.user
 
-        user_in_valid_group = user.is_ofa_sys_admin
+        user_in_valid_group = user.is_ofa_sys_admin or user.is_digit_team
```

Comment:
i added some logging because i wasn't sure if i was missing something, but the kibana auth check endpoint seems to allow access even if the user is not allowed and `BYPASS_KIBANA_AUTH` is `False`
<img width="516" alt="image" src="https://github.com/raft-tech/TANF-app/assets/111305129/138be570-ced3-4f94-9bb7-26b56109cc69">
`hhs_id` is also `None` in this example but i didn't print that

Reply:
i tested several values for `BYPASS_KIBANA_AUTH` in the `.env` file, including `False`, `0`, `no`, and `"no"` . The only thing that worked was commenting it out entirely and letting the default value from the settings file take over. i think this is working as expected, but a suggestion would be to use `strtobool()` similar to how we do for the `DJANGO_DEBUG` setting.